modules/host/consts.go

```diff
@@ -104,6 +131,38 @@ var (
 	// data.
 	defaultUploadBandwidthPrice = types.SiacoinPrecision.Mul64(10).Div(modules.BytesPerTerabyte) // 10 SC / TB
 
+	// workingStatusFrequency defines how frequently the Host's working status
+	// check runs
+	workingStatusFrequency = build.Select(build.Var{
+		Standard: time.Minute * 15,
+		Dev:      time.Minute * 5,
+		Testing:  time.Second * 30,
+	}).(time.Duration)
+
+	// workingStatusThreshold defines how many settings calls must occur over the
+	// workingStatusFrequency for the host to be considered working.
+	workingStatusThreshold = build.Select(build.Var{
+		Standard: uint64(3),
+		Dev:      uint64(1),
+		Testing:  uint64(1),
+	}).(uint64)
```

Comment:
Sometimes the number will increment because you are pinging yourself. It's not supposed to happen, but legacy users that upgrade to this functionality may have themselves in the hostdb via a local route.